core/config/general_config.go

```diff
@@ -813,7 +816,7 @@ func (c *generalConfig) OCRBootstrapCheckInterval() time.Duration {
 }
 
 func (c *generalConfig) OCRContractTransmitterTransmitTimeout() time.Duration {
```

Comment:
These functions can be removed from here and from the general config interface since you must use the chain-specific versions now

Reply:
Done. I hope I did this right this time.